hosted/json-schemas/message.schema.json

```diff
@@ -11,7 +11,7 @@
           "description": "The sender's DID"
         },
         "to": {
-          "$ref": "https://tbdex.dev/definitions.json#/definitions/did",
+          "$ref": "definitions.json#/definitions/did",
```

Comment:
changed to be consistent with how we use $ref elsewhere https://github.com/TBD54566975/tbdex/pull/247/files#diff-781d77c475148a35130d81220b3a6d100357fc53576806be6dda9a528f1ae283L10